roles/keystone/defaults/main.yml

```diff
@@ -26,3 +29,7 @@ keystone:
   logging:
     debug: False
     verbose: True
+
+  ldap:
+    url: ldap://127.0.0.1:389
```

Comment:
This is role default, I tried to set a reasonable one considering we don't know what the final environment will be. This is properly overridden in envs/example/defaults.yml as below

  ldap:<br>
    url: <b>ldap://{{ undercloud_floating_ip }}:390</b>